src/protocol/prss.rs

```diff
@@ -35,6 +85,11 @@ impl IndexedSharedRandomness {
     #[must_use]
     pub fn generate_values<I: Into<u128>>(&self, index: I) -> (u128, u128) {
         let index = index.into();
+        #[cfg(debug_assertions)]
+        {
+            self.used.insert(index);
+        }
```

Comment:
```suggestion
        if cfg!(debug_assertions) {
            self.used.insert(index);
        }
```

Reply:
that does not work for release builds as `used` field is in debug only. we have to take out the whole block for release